### PR TITLE
Add instructions for running multiple wallets on the same machine

### PR DESCRIPTION
**Pull request by makeasnek**
Instructions for running multiple wallets on the same machine

### CODE REVIEW

File: wiki/faq.md [hunk 1]
```diff
@@ -529,6 +529,24 @@ try to pull whatever it can out of the wallet.dat file and will create a new
 file. Rename that new file to wallet.dat and try launching the wallet again
 without the `-salvagewallet` flag
 
+---
+### How can I run multiple wallets on the same machine?
+
+You should never run the same wallet (start the same wallet.dat) on multiple machines. But if you desire multiple wallets on the same machine, such as for identity seperation, it is possible. You will just need to launch Gridcoin with the arguments -datadir="/path/to/your/datadir" and give each seperately launched Gridcoin instance a directory to itself.
```
Comment (barton2526) [inline wiki/faq.md:535]: Separation

File: wiki/faq.md [hunk 2]
```diff
@@ -529,6 +529,24 @@ try to pull whatever it can out of the wallet.dat file and will create a new
 file. Rename that new file to wallet.dat and try launching the wallet again
 without the `-salvagewallet` flag
 
+---
+### How can I run multiple wallets on the same machine?
+
+You should never run the same wallet (start the same wallet.dat) on multiple machines. But if you desire multiple wallets on the same machine, such as for identity seperation, it is possible. You will just need to launch Gridcoin with the arguments -datadir="/path/to/your/datadir" and give each seperately launched Gridcoin instance a directory to itself.
+
+You can then add entries to your system's startup manager to have these seperate copies run at startup.
```
Comment (barton2526) [inline wiki/faq.md:537]: separate

Comment (makeasnek) [reply wiki/faq.md]: Good catch fixed

File: wiki/faq.md (hunk 1 above)
Comment (makeasnek) [reply wiki/faq.md]: Good catch fixed

**Pull request closed without merging**